rust_src/src/cmds.rs

```diff
@@ -341,7 +338,7 @@ fn internal_self_insert(mut c: Codepoint, n: usize) -> EmacsInt {
         // or before a tab if it doesn't use the whole width of the tab.  */
         if overwrite == Qoverwrite_mode_binary {
             chars_to_delete = n as usize;
-        } else if c != '\n' as Codepoint && c2 != '\n' as Codepoint {
+        } else if c != Codepoint::from('\n') && c2 != Codepoint::from('\n') {
```

Comment:
Are the conversion needed here?

Reply:
Nope, that's a leftover from before i implented `PartialEq<char>`